constitution/15-retirement_and_removal_of_charity_trustees.tex

```diff
@@ -2,10 +2,10 @@
 \section{Retirement and Removal of Charity Trustees}\label{sec:retirement_removal}
 
     \subsection{}\label{sec:retirement}
-    A charity trustee ceases to hold office if he or she:
+    A charity trustee ceases to hold office if they:
     \begin{enumerate}
         \item retires by notifying \shortname{} in writing (but only if enough charity trustees will remain in office when the notice of resignation takes effect to form a quorum for meetings);
```

Comment:
“if they retires”

I need to review this more carefully before putting it to the members. Not much time before the 2018 AGM! 😉